### PR TITLE
Add right sign currency support in currency converter

### PR DESCRIPTION
**Pull request by ademar111190**
Fixes https://github.com/breez/breezmobile/issues/760

How it looks like:

https://user-images.githubusercontent.com/1225438/163410546-289ec84a-b22f-4fd0-a523-79f7026e7dc8.mp4

### CODE REVIEW

**Comment by tlindi** [comment]
TY 💯

**Comment by ademar111190** [comment]
I did apply the masking strategy, I did update the description with a video of what it looks like. please take a look @kingonly

**Comment by kingonly** [comment]
@ademar111190 this is perfect!

**Review by roeierez — CHANGES_REQUESTED** [review]
Very nice!

File: lib/widgets/currency_amount_field_formatter.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    try {
+      double.parse(raw);
+    } catch (_) {
+      return oldValue;
+    }
+
+    if (raw.contains(".")) {
+      if (raw.split(".").last.length > currencyData.fractionSize) {
+        return oldValue;
```
[the remark this answers is not in the document]
Comment (ademar111190) [reply lib/widgets/currency_amount_field_formatter.dart]: You are right it is better, I did apply it.

**Review by roeierez — APPROVED** [review]

**Pull request merged**